i2c/oled_i2c/raspberry26x32.h

```diff
@@ -0,0 +1,4 @@
+#define IMG_WIDTH 26
+#define IMG_HEIGHT 32
+
+static uint8_t raspberry26x32[] = { 0x0, 0x0, 0xe, 0x7e, 0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfe, 0xfe, 0xfc, 0xf8, 0xfc, 0xfe, 0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfe, 0x7e, 0x1e, 0x0, 0x0, 0x0, 0x80, 0xe0, 0xf8, 0xfd, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfd, 0xf8, 0xe0, 0x80, 0x0, 0x0, 0x1e, 0x7f, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x7f, 0x1e, 0x0, 0x0, 0x0, 0x3, 0x7, 0xf, 0x1f, 0x1f, 0x3f, 0x3f, 0x7f, 0xff, 0xff, 0xff, 0xff, 0x7f, 0x7f, 0x3f, 0x3f, 0x1f, 0x1f, 0xf, 0x7, 0x3, 0x0, 0x0};
```

Comment:
Was this generated by some external tool? Might be nice to add details to the README so that users can use their own logo if they want?

Reply:
I wrote a quick Python script and it did the job but it's not very pretty xD

Reply:
It's an SVG(?) image converted to Base 64, right? In which case there's almost certainly an online tool we can link to, some of them might even be hosted somewhere we'd want to (be willing to) link to...

Reply:
AFAICT it's an SVG image rendered out to a 26*32 monochrome bitmap, bit-swizzled to the page/column formatting suitable for the SSD1306 chip, and then encoded as a C-array of hex-bytes?

Reply:
That might be harder to find an online tool for… 🤔

Reply:
Would it be worth touching it up and including it in the folder?

Reply:
> AFAICT it's an SVG image rendered out to a 26*32 monochrome bitmap, bit-swizzled to the page/column formatting suitable for the SSD1306 chip, and then encoded as a C-array of hex-bytes?

Yes, it's exactly that!

Reply:
> Yes, it's exactly that!

In which case I think it's worth beautify'ing the Python script and including it along with the example, and add it in the 'list of files' so it gets pulled into the docs. I'm crossing my fingers it doesn't use any complicated tomfoolery and it's all just default installed Python modules so folks don't have to mess with their Python installation to run it (i.e. you have Python, the script runs)..?

Reply:
Almost..It uses Pillow which is not in the standard library but is almost as universal as the likes of `matplotlib` and `numpy`, I think?

Reply:
Yeah, Pillow is fairly standard. Brief pip install installation instructions in the comments at the top should be enough for that one.

Reply:
Done